### PR TITLE
Feature/kniktas cleanup

### PR DESCRIPTION
**Pull request by kostasniktas**

### CODE REVIEW

**Comment by kostasniktas** [comment]
@EXBreder @davidk01

File: src/main/groovy/com/palantir/gradle/versions/flexversioning/FlexVersion.groovy [hunk 1]
```diff
@@ -23,6 +29,7 @@ class FlexVersion {
     boolean dirty = false;
     boolean tag = false;
 
+
     public FlexVersion (String domain, int commitCount, String gitHash, boolean tag, boolean dirty) {
```
Comment (imawuman) [inline src/main/groovy/com/palantir/gradle/versions/flexversioning/FlexVersion.groovy:32]: my only compliant is the fullVersion variable existing, but you already know that :). Other than that +1 lgtm

**Comment by uschi2000** [comment]
Sweet, thanks.

**Comment by imawuman** [comment]
+1

**Pull request merged**